operator/api/server.go

```diff
@@ -106,9 +112,23 @@ func (s *Server) StartServer() error {
 			continue
 		}
 		nServers++
+
+		mux := http.NewServeMux()
+
+		// Index handler is the the handler for Open-API router.
+		mux.Handle("/", s.Server.GetHandler())
+		// Create a custom handler for /healthz as an alias to /v1/healthz. A http mux
+		// is required for this because open-api spec does not allow multiple base paths
+		// to be specified.
+		mux.HandleFunc("/healthz", func(rw http.ResponseWriter, _ *http.Request) {
+			resp := s.healthzHandler.Handle(operator.GetHealthzParams{})
+			rw.Header().Set("Content-Type", "application/json")
```

Comment:
I was almost missing this, setting the header in here made me double check. The current master is simply returning `ok` with content-type `plain/text`. This healthz should not be modified in the `/healthz` nor `/v1/healthz`.

```
$ curl -vvv 127.0.0.1:9234/healthz
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to 127.0.0.1 (127.0.0.1) port 9234 (#0)
> GET /healthz HTTP/1.1
> Host: 127.0.0.1:9234
> User-Agent: curl/7.58.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Date: Tue, 05 Jan 2021 16:49:13 GMT
< Content-Length: 2
< Content-Type: text/plain; charset=utf-8
< 
* Connection #0 to host 127.0.0.1 left intact
ok
```

Reply:
I have pushed the required changes to keep the schema the same as before. 
![image](https://user-images.githubusercontent.com/21292343/103748600-60ee2380-502a-11eb-9e19-bf9fa0248e9b.png)